### ISSUE TITLE
Additional css classes for question/panel

### ISSUE THREAD

**Issue by notz** [surveyjs/survey-creator#1041]
### Are you requesting a feature, reporting a bug or ask a question?

Feature

### What is the expected behavior?

Currently in the creator there is no option (except of abusing the name) which can be used to style a question / panel differently.
I would love to see an additional input field to enter a css class or classes to support styling without changing the javascript code or abusing the name of the question / panel.

**Comment by tsv2013** [comment]
If you want to style questions/panels in a designed survey, the question is related to SurveyJS Library. And we have corresponding example on our site - https://surveyjs.io/Examples/Library?id=survey-cssclasses&platform=jQuery&theme=bootstrap#content-result

**Comment by notz** [comment]
Yes, i'm aware of that. But that's not what i have requested. I like a field in creator to style for example a text field with a red background an another text field with a green background without modifying the javascript.

I suggest to add a an Text Field in the Layout Section of each question / panel named "Style", this is added to class definitions of the question on rendering.

Style: "red"

`<div class="sv_qstn red" ...`

```
.sv_qstn .red {
  background-color: red;
}
.sv_qstn .green {
  background-color: green;
}
```

Otherwise the only chance is to abuse the name (there is no other free text field) with an rather complex css selector:

`[name*=".red."]`

**Comment by tsv2013** [comment]
You task can be achieved via custom code. We don't want to add CSS or something similar into survey JSON model due to have plans to implement native iOS/Android survey JSON rendering.

**Comment by notz** [comment]
Yes, but not with a shared custom code across different surveys. Makes no sense if you need a custom code for every survey created with the survey creator. The main advantage of the creator is that you don't need a developer to create the survey. A additional text field will be enough to solve the case and will also be very helpful to create Custom Code that shared across different surveys.

**Comment by tsv2013** [comment]
>Yes, but not with a shared custom code across different surveys. 
You can run the same code for each survey created on your site or in your application.

You can add a custom property, users will fill this property in survey creator, your code during runtime will get value(s) from the custom property and add corresponding styles. That's the brief implemantation idea.
